Index: 9base-6/rc/Makefile
===================================================================
--- 9base-6.orig/rc/Makefile
+++ 9base-6/rc/Makefile
@@ -32,14 +32,14 @@ install: ${TARG}
 	@mkdir -p ${DESTDIR}${MANPREFIX}/man1
 	@cp -f ${MANFILES} ${DESTDIR}${MANPREFIX}/man1
 	@chmod 644 ${DESTDIR}${MANPREFIX}/man1/${MANFILES}
-	@mkdir -p ${DESTDIR}${PREFIX}/etc
-	@cp -f rcmain ${DESTDIR}${PREFIX}/etc
-	@chmod 755 ${DESTDIR}${PREFIX}/etc/rcmain
+	@mkdir -p ${DESTDIR}/etc
+	@cp -f rcmain ${DESTDIR}/etc
+	@chmod 755 ${DESTDIR}/etc/rcmain
 
 uninstall:
 	rm -f ${DESTDIR}${PREFIX}/bin/${TARG}
 	rm -f ${DESTDIR}${MANPREFIX}/man1/${MANFILES}
-	rm -f ${DESTDIR}${PREFIX}/etc/rcmain
+	rm -f ${DESTDIR}/etc/rcmain
 
 .c.o:
 	@echo CC $*.c
